### PR TITLE
Feature/bi 3883 landing page

### PR DESCRIPTION
**Pull request by mattch1**
Just added a route and template as the cdn styling and parent templates are being done on other tickets.

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            Object to a company being struck off
+        </h1>
+
+        <p>Use this service to tell us why a limited company should not be removed from the companies register.
+           You can only do this after the notice has been posted in The Gazette.</p>
+        <p>Companies House must receive the objection at least 2 weeks before the notice expiry date.</p>
+
+        <p>This service takes around 15 minutes.</p>
+
+        <a href="/sign-in" role="button" draggable="false" class="govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8" data-module="govuk-button">
```
Comment (ICoxCH) [inline views/index.html:19]: The sign-in page is going to be achieved via an interceptor, so you'll just need to put the next page in here, not a link to sign-in

Comment (mattch1) [reply views/index.html]: Using link to company number page instead.

File: src/routes/routes.ts [hunk 2]
```diff
@@ -1,7 +1,11 @@
-import { Router } from "express";
+import { NextFunction, Request, Response, Router } from "express";
 
 const router: Router = Router();
 
-router.get("/", (req, response) => response.send("This is the Strike Off Objections app"));
+const renderTemplate = (template: string) => (req: Request, res: Response, next: NextFunction) => {
+    return res.render(template, { templateName: template });
```
Comment (mwestacott) [inline src/routes/routes.ts:6]: Presume we're still using the same formatting standard as PTF - 2 spaces for indents? If so, the indent here will need amending.

Comment (mattch1) [reply src/routes/routes.ts]: Ammended.

**Pull request merged**